README.md

```diff
@@ -125,6 +146,31 @@ Unstable options are not available on stable/beta with Rustfmt v1.x.
 
 See the configuration documentation on the Rustfmt [GitHub page](https://rust-lang.github.io/rustfmt/) for details (look for the `unstable_features` section).
 
+### Differences in rustfmt versions
+
+Aside differences in [configuration options](#configuring-rustfmt), versions of rustfmt may differ
+in formatting behavior.
+
```

Comment:
This sentence could pretty easily be interpreted as contradicting the stability guarantee in its current form (#4286), so we'll need to remove or reword this. Based on feedback from the dev tools team and some folks from the core team, I believe that it's possible some changes may be coming that could directly impact what sort of differences may appear between differing kinds of new rustfmt versions (major vs. minor/patch).

For now, if you think there's value in having some text under this heading, I'd suggest something more narrow and explicit such as:

```
New [configuration options](#configuring-rustfmt) may be introduced in rustfmt releases, and versions of rustfmt may differ in formatting behavior of new syntax, with newer versions of rustfmt having better formatting support for new syntax.
```

Reply:
I just got rid of the sentence.

Reply:
Hmm. Original sentence still seems to be here?

Reply:
): I forgot to push. Sorry about that, will ping you later..

Reply:
Lol, no worries. Will merge after that :+1:

Reply:
done, ptal